lib/vets/collections/cacheable.rb

```diff
@@ -0,0 +1,77 @@
+# frozen_string_literal: true
+
+require 'active_support/concern'
+
+module Vets
+  module Collections
+    module Cacheable
+      extend ActiveSupport::Concern
+
+      CACHE_NAMESPACE = 'common_collection'
+      CACHE_DEFAULT_TTL = 3600 # default to 1 hour
+
+      class_methods do
+        # rubocop:disable ThreadSafety/ClassInstanceVariable
+        def redis_namespace
+          @redis_namespace ||= Redis::Namespace.new(CACHE_NAMESPACE, redis: $redis)
```

Comment:
This should probably be moved to a constant instantiated at load/boot.

Alternatively it could be made thread safe via:

```ruby
Thread.current[:vets_cacheable_ns] ||= Redis::Namespace.new(CACHE_NAMESPACE, redis: $redis)
```